astro.config.mjs

```diff
@@ -35,6 +36,235 @@ export default defineConfig({
+										badge: 'Coming soon',
+									},
+								],
+							},
+							{
+								label: 'Maintaining',
+								collapsed: true,
+								items: [
+									{
+										label: 'Introduction',
+										slug: 'guide/maintaining-open-source-projects',
+									},
+									{
+										slug: 'guide/maintaining-open-source-projects/introduction-to-open-source-project-maintenance',
+									},
+									{
+										slug: 'guide/maintaining-open-source-projects/managing-contributions-and-community-engagement',
+										badge: 'Coming soon',
+									},
+									{
+										slug: 'guide/maintaining-open-source-projects/managing-project-dependencies',
+										badge: 'Coming soon',
+									},
+									{
+										slug: 'guide/maintaining-open-source-projects/fostering-a-strong-and-inclusive-community',
+									},
+									{
+										slug: 'guide/maintaining-open-source-projects/ensuring-project-sustainability',
+										badge: 'Coming soon',
+									},
+								],
+							},
+							{
+								label: 'Promoting',
+								collapsed: true,
+								items: [
+									{
+										label: 'Introduction',
+										slug: 'guide/promoting-open-source-projects',
+									},
+									{
+										slug: 'guide/promoting-open-source-projects/introduction-to-project-promotion',
+									},
+									{
+										slug: 'guide/promoting-open-source-projects/building-a-strong-project-identity',
+									},
+									{
+										slug: 'guide/promoting-open-source-projects/crafting-an-engaging-project-website',
+									},
+								],
+							},
+							{
+								label: 'Financing',
+								collapsed: true,
+								items: [
+									{
+										label: 'Introduction',
+										slug: 'guide/financing-open-source-projects',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/importance-and-challenges-of-financing-open-source-projects',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/understanding-funding-models',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/effective-fundraising-strategies',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/resource-allocation-and-budgeting',
+										badge: 'Coming soon',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/fostering-a-sustainable-ecosystem',
+										badge: 'Coming soon',
+									},
+									{
+										slug: 'guide/financing-open-source-projects/transparency-accountability-and-community-involvement',
+										badge: 'Coming soon',
+									},
+								],
+							},
+						],
+          },
+					{
+						label: 'Articles',
+            link: '/articles/',
+            icon: 'seti:odata',
+						items: [
+							{
+								label: 'Articles',
+								link: '/articles',
+							}
+						]
+					},
+					{
+						label: 'Resources',
+            link: '/resources/books',
+            icon: 'seti:svg',
+						items: [
```

Comment:
For now, did not manage to use `autogenerate` here. The best I could find so far was:

```js
{
	label: 'Resources',
	link: '/resources/books',
	icon: 'seti:svg',
	items: [
		{
			label: '',
			autogenerate: { directory: 'resources' },
		}
	]
},
```